v19.2/performance.md

```diff
@@ -0,0 +1,101 @@
+
+<!-- To try this out for yourself on your laptop [visit this docs page](https://www.cockroachlabs.com/docs/v19.1/training/performance-benchmarking.html#main-content). Or, if you want to reproduce the full results of TPC-C [visit this page](https://www.cockroachlabs.com/guides/cockroachdb-performance/). -->
+
+### Linear Scaling
+
+CockroachDB has **no theoretical scaling limit** and, in practice, can achieve near-linear performance passed more than 256 nodes. Because the TPC-C results above reflect leaps in scale, to test linear scaling, Cockroach Labs ran a simple benchmark named KV 95 (95% point reads, 5% point writes, all uniformly distributed) on AWS `c5d.4xlarge` machines:
+
+<img src="{{ 'images/v19.2/linearscale.png' | relative_url }}" alt="CRDB Linear Scale" style="max-width:100%" />
+
+This chart shows that adding nodes increases throughput linearly while holding p50 and p99 latency constant.
+
+## Throughput
+
+As mentioned above, Cockroach Labs believes TPC-C provides the most realistic and objective measure for OLTP throughput. In the real world, applications generate transactional workloads that consist of a combination of reads and writes, possibly with concurrency and likely without all data being loaded into memory. If you see benchmark results quoted in QPS, take them with a grain of salt, because anything as simple as a “query” is unlikely to be representative of the workload you need to run in practice.
+
+With that in mind, however, you can use [Sysbench](https://github.com/akopytov/sysbench) for straight-forward throughput benchmarking. For example, on a 3-node cluster of AWS `c5d.9xlarge` machines across AWS’s `us-east-1` region (availability zones `a`, `b`, and `c`), CockroachDB can achieve 118,000 inserts per second on the `oltp_insert` workload and 336,000 reads per second on the `oltp_point_select` workload:
+
+<img src="{{ 'images/v19.2/sysbench-throughput.png' | relative_url }}" alt="Sysbench Throughput" style="max-width:100%" />
+
+## Latency
+
+CockroachDB returns single-row **reads in 1 ms or less** and processes single-row **writes in 2 ms or less**, with a number of important tuning practices for both single-region and multi-region deployments, including [secondary indexes](indexes.html) and various [data topologies](topology-patterns.html).
+
+For benchmarking latency, again, Cockroach Labs believes TPC-C provides the most realistic and objective measure, since it encompasses the latency distribution, including tail performance. However, you can use [Sysbench](https://github.com/akopytov/sysbench) for straight-forward latency benchmarking. For example, on a 3-node cluster of AWS `c5d.9xlarge` machines across AWS’s `us-east-1` region (availability zones `a`, `b`, and `c`), CockroachDB can achieve an average of 4.3ms on the `oltp_insert` workload and 0.7ms on the `oltp_point_select` workload:
+
+<img src="{{ 'images/v19.2/sysbench-latency.png' | relative_url }}" alt="Sysbench Latency" style="max-width:100%" />
```

Comment:
This image really badly needs to explicitly call out what that value represents. 99th percentile? It should be in your face. We also should call out the concurrency of the workload.

Reply:
@nvanbenschoten let's connect on monday because i don't have the raw data

Reply:
Average latency and we will update the chart. Note, we will p95 in the future as Sysbench provides it but we don't have it right now without re-running so we can add it in a future version if we think it is valuable